frontend/ts/index.ts

```diff
@@ -13,7 +13,7 @@ import hd from "humanize-duration";
 })();
 
 // Set timezone
-(function () {
+(() => {
```

Comment:
Same here

Reply:
@natsukagami it seems that both work just fine, so unifying the code style would be nice.